lib/vexml/types.ts

```diff
@@ -16,27 +16,25 @@ export type BeamEndMessage = {
 
 export type NoteMessage = {
   type: 'note';
-  stem: 'up' | 'down';
```

Comment:
This and the other string type changes: I think this is ok for now, just to get something working, but we really should be more specific when we can. We can create type asserts like this:

```ts
type Stem = 'up' | 'down';
const isStem = (str: string): str is Stem => ['up', 'down'].includes(str);
```

Then, in the call sites, we can assert the type and throw an error (or something else) if we get something unexpected. What do you think?

Reply:
This is a good idea. However once again, we have to consider if we want to use Node directly. What do you think?

Reply:
Let's talk about `Node` in this discussion: https://github.com/jaredjj3/vexml/pull/5#discussion_r835290896

For now, let's leave it as is and just keep that in mind if/when we want to productionize this.